docs/src/cfe_api.dox

```diff
@@ -0,0 +1,268 @@
+    <UL>
+      <LI> #CFE_ES_RegisterApp - \copybrief CFE_ES_RegisterApp
+      <LI> #CFE_ES_RunLoop - \copybrief CFE_ES_RunLoop
+      <LI> #CFE_ES_WaitForStartupSync - \copybrief CFE_ES_WaitForStartupSync
+      <LI> #CFE_ES_WaitForSystemState - \copybrief CFE_ES_WaitForSystemState
+      <LI> #CFE_ES_IncrementTaskCounter - \copybrief CFE_ES_IncrementTaskCounter
+      <LI> #CFE_ES_ExitApp - \copybrief CFE_ES_ExitApp
+    </UL>
+    <LI> \ref CFEAPIESInfo
+    <UL>
+      <LI> #CFE_ES_GetResetType - \copybrief CFE_ES_GetResetType
+      <LI> #CFE_ES_GetAppID - \copybrief CFE_ES_GetAppID
+      <LI> #CFE_ES_GetAppIDByName - \copybrief CFE_ES_GetAppIDByName
+      <LI> #CFE_ES_GetAppName - \copybrief CFE_ES_GetAppName
+      <LI> #CFE_ES_GetAppInfo - \copybrief CFE_ES_GetAppInfo
+      <LI> #CFE_ES_GetTaskInfo - \copybrief CFE_ES_GetTaskInfo
+    </UL>
+    <LI> \ref CFEAPIESChildTask
+    <UL>
+      <LI> #CFE_ES_RegisterChildTask - \copybrief CFE_ES_RegisterChildTask
+      <LI> #CFE_ES_CreateChildTask - \copybrief CFE_ES_CreateChildTask
+      <LI> #CFE_ES_DeleteChildTask - \copybrief CFE_ES_DeleteChildTask
+      <LI> #CFE_ES_ExitChildTask - \copybrief CFE_ES_ExitChildTask
+    </UL>
+    <LI> \ref CFEAPIESCritData
+    <UL>
+      <LI> #CFE_ES_RegisterCDS - \copybrief CFE_ES_RegisterCDS
+      <LI> #CFE_ES_CopyToCDS - \copybrief CFE_ES_CopyToCDS
+      <LI> #CFE_ES_RestoreFromCDS - \copybrief CFE_ES_RestoreFromCDS
+    </UL>
+    <LI> \ref CFEAPIESMemManage
+    <UL>
+      <LI> #CFE_ES_PoolCreate - \copybrief CFE_ES_PoolCreate
+      <LI> #CFE_ES_PoolCreateEx - \copybrief CFE_ES_PoolCreateEx
+      <LI> #CFE_ES_PoolCreateNoSem - \copybrief CFE_ES_PoolCreateNoSem
+      <LI> #CFE_ES_GetPoolBuf - \copybrief CFE_ES_GetPoolBuf
+      <LI> #CFE_ES_PutPoolBuf - \copybrief CFE_ES_PutPoolBuf
+      <LI> #CFE_ES_GetMemPoolStats - \copybrief CFE_ES_GetMemPoolStats
+      <LI> #CFE_ES_GetPoolBufInfo - \copybrief CFE_ES_GetPoolBufInfo
+    </UL>
+    <LI> \ref CFEAPIESPerfMon
+    <UL>
+      <LI> #CFE_ES_PerfLogEntry - \copybrief CFE_ES_PerfLogEntry
+      <LI> #CFE_ES_PerfLogExit - \copybrief CFE_ES_PerfLogExit
+      <LI> #CFE_ES_PerfLogAdd - \copybrief CFE_ES_PerfLogAdd
+    </UL>
+    <LI> \ref CFEAPIESGenCount
+    <UL>
+      <LI> #CFE_ES_RegisterGenCounter - \copybrief CFE_ES_RegisterGenCounter
+      <LI> #CFE_ES_DeleteGenCounter - \copybrief CFE_ES_DeleteGenCounter
+      <LI> #CFE_ES_IncrementGenCounter - \copybrief CFE_ES_IncrementGenCounter
+      <LI> #CFE_ES_SetGenCount - \copybrief CFE_ES_SetGenCount
+      <LI> #CFE_ES_GetGenCount - \copybrief CFE_ES_GetGenCount
+      <LI> #CFE_ES_GetGenCounterIDByName - \copybrief CFE_ES_GetGenCounterIDByName
+    </UL>
+    <LI> \ref CFEAPIESMisc
+    <UL>
+      <LI> #CFE_ES_CalculateCRC - \copybrief CFE_ES_CalculateCRC
+      <LI> #CFE_ES_WriteToSysLog - \copybrief CFE_ES_WriteToSysLog
+      <LI> #CFE_ES_ProcessCoreException - \copybrief CFE_ES_ProcessCoreException
```

Comment:
Missed a change here in your conflict resolution.  You need to carry over the change that happened in the IC branch.